### PR TITLE
Fix for the broken synchronised rendering

### PR DESCRIPTION
**Pull request by bilgili**

### CODE REVIEW

**Comment by jafyvilla** [comment]
Could you extend the commit description a bit more? It's hard to know what the problem was before.

**Comment by bilgili** [comment]
@jafyvilla : I think we should add those tests but I am not seeing an easy way to add them.

**Comment by bilgili** [comment]
@jafyvilla : I will check for the data source, I was not sure which code is being run first. It is not very clear.

**Comment by jafyvilla** [comment]
I'm testing this PR and 'livre' (no arguments) doesn't render anything

File: livre/eq/Node.cpp [hunk 1]
```diff
@@ -119,17 +119,36 @@ class Node
...
 
+    void updateAndSendFrameRange()
+    {
+        if( !_dataSourcePtr )
+            return;
+
+        Vector2ui prevRange;
+        Vector2ui currentRange;
+        getCurrentPrevFrameRanges( prevRange, currentRange );
+        if( currentRange != prevRange )
```
Comment (jafyvilla) [inline livre/eq/Node.cpp]: I'm testing and I need this here
"if( currentRange != prevRange || currentRange == FULL_FRAME_RANGE )"
to have the default 'livre' working

**Comment by jafyvilla** [comment]
:+1:

**Pull request merged**